ruby/vampires.rb

```diff
@@ -0,0 +1,66 @@
+# Vampire Tester
+
+applicant_status = nil
+current_year = 2017

```

Comment:
Check out the Ruby Date module so you could dynamically change the year. You can call `Date.now` and extract the year from that.